packages/vite/src/node/plugins/reporter.ts

```diff
@@ -32,6 +32,7 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
   let transformedCount = 0
   let chunkCount = 0
   let compressedCount = 0
+  let startTime = +new Date()
```

Comment:
I think the usual pattern in the codebase is using `Date.now()` instead. It's nicer too since you get a number directly.